hub3/ead/mets.go

```diff
@@ -397,144 +284,31 @@ func findingAidTriples(subject string, fa *eadpb.FindingAid) []*rdf.Triple {
-		CLevel:      fa.InventoryPath,
-		UnitID:      fa.InventoryID,
-		MimeTypes:   []string{file.MimeType},
-		SortKey:     uint64(file.SortKey),
-		Title:       fa.InventoryTitle,
-		InventoryID: fa.ArchiveID,
-		Label:       file.Filename,
-	}
-
-	b, err := proto.Marshal(file)
-	if err != nil {
-		return fg, fmt.Errorf("unable to marshal protobuf message: %#v", err)
-	}
-
-	fg.ProtoBuf = &fragments.ProtoBuf{
-		MessageType: "eadpb.File",
-		Data:        fmt.Sprintf("%x", b),
-	}
-	fg.SetResources(rm)
-
-	return fg, nil
-}
-
-func createHeader(cfg *NodeConfig, id, subjectBase, tag string) *fragments.Header {
-	subject := fmt.Sprintf("%s/%s", subjectBase, id)
-	header := &fragments.Header{
-		OrgID:    cfg.OrgID,
-		Spec:     cfg.Spec,
-		Revision: cfg.Revision,
-		HubID: fmt.Sprintf(
-			"%s_%s_%s",
-			cfg.OrgID,
-			cfg.Spec,
-			strings.ReplaceAll(id, "/", "-"),
-		),
-		DocType:       fragments.FragmentGraphDocType,
-		EntryURI:      subject,
-		NamedGraphURI: fmt.Sprintf("%s/graph", subject),
-		Modified:      fragments.NowInMillis(),
-		Tags:          []string{tag},
-	}
-
-	return header
-}
-
 // saveFileFragmentGraphs saves all eadpb.File and eadpb.FindingAid graphs to ElasticSearch.
 // Note that during the process the files are removed from the eadpb.FindingAid.
-func saveFileFragmentGraphs(cfg *NodeConfig, fa *eadpb.FindingAid) error {
-	for _, file := range fa.Files {
-		fg, err := fragmentGraph(cfg, fa, file)
-		if err != nil {
-			return err
-		}
-
-		m, err := fg.IndexMessage()
-		if err != nil {
-			return err
-		}
-
-		if cfg.IndexService != nil {
-			cfg.IndexService.Publish(context.Background(), m)
-			atomic.AddUint64(&cfg.RecordsPublishedCounter, 1)
-		}
-	}
-
-	fg, err := findingAidFragmentGraph(cfg, fa)
-	if err != nil {
-		return err
-	}
-
-	m, err := fg.IndexMessage()
-	if err != nil {
-		return err
-	}
-
-	if cfg.IndexService != nil {
-		cfg.IndexService.Publish(context.Background(), m)
-		atomic.AddUint64(&cfg.RecordsPublishedCounter, 1)
-	}
-
-	return nil
-}
-
-func findingAidFragmentGraph(cfg *NodeConfig, fa *eadpb.FindingAid) (*fragments.FragmentGraph, error) {
-	// remove files because we don't want them to be saved in bbolt
-	fa.Files = []*eadpb.File{}
-
-	subjectBase := fmt.Sprintf("%s/NL-HaNA/archive/%s/%s", c.Config.RDF.BaseURL, fa.ArchiveID, fa.InventoryID)
-	id := fmt.Sprintf("%s-findingaid", fa.GetInventoryID())
-	header := createHeader(cfg, id, subjectBase, "findingaid")
-
-	rm := fragments.NewEmptyResourceMap()
-
-	for idx, t := range findingAidTriples(header.EntryURI, fa) {
-		if err := rm.AppendOrderedTriple(t, false, idx); err != nil {
-			return nil, err
-		}
-	}
-
-	fg := fragments.NewFragmentGraph()
-	fg.Meta = header
-	fg.Tree = &fragments.Tree{
-		CLevel:      fa.InventoryPath,
-		UnitID:      fa.InventoryID,
-		Title:       fa.InventoryTitle,
-		InventoryID: fa.ArchiveID,
-		Label:       "findingaid",
-	}
+// func saveFileFragmentGraphs(cfg *NodeConfig, fa *eadpb.FindingAid) error {
```

Comment:
From this point onward, do we need all commented code in the future?

Reply:
No, but I want to clean this up after the package `hub3/models` is removed.